planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/configschema/FeatureGeometryType.java

```diff
@@ -0,0 +1,7 @@
+package com.onthegomap.planetiler.custommap.configschema;
+
+public enum FeatureGeometryType {
+  polygon,
+  linestring,
+  point
```

Comment:
We already have a `com.onthegomap.planetiler.geo.GeometryType` enum for these 3 geometry types, but I'm also wondering how we'll handle things like polygon centroids? If this will grow based on different kinds of geometry transformations we might do then it's fine to have separate.

If you're going to keep here, then we can also still use conventional upper-camel case but preserve the yaml representations by using the `@JsonNode` annotation, for example:

https://github.com/onthegomap/planetiler/blob/6a7c6791416c06fe6bbbcdebaf74cf078e1c2ea4/planetiler-core/src/main/java/com/onthegomap/planetiler/mbtiles/Mbtiles.java#L265-L271

Reply:
Resolved in 896f3cd